firmware/boron-ins-fsm/src/imDoorSensor.cpp

```diff
@@ -175,7 +175,7 @@ void threadBLEScanner(void *param) {
     filter.address(address);
 
     // scanning for IM21 and IM24 door sensors of correct device ID
-    Vector<BleScanResult> scanResults = BLE.scanWithFilter(filter);
+    spark::Vector<BleScanResult> scanResults = BLE.scanWithFilter(filter);
```

Comment:
I noticed that the compiler says that "Vector is not declared in this scope" when the spark:: representing the namespace is not included.

Reply:
OK, interesting. I think I get it. When this is compiled in real life, it'll have all the spark headers, including `firmware/boron-ins-fsm/inc/spark_wiring_vector.h`, so it would use that automatically. But since there also seems to be a `std::Vector` and one defined in Catch2, it gets confused. Maybe.